### PR TITLE
SDK-371 - Ensure all frontend configuration is made available to fron…

### PR DESCRIPTION
**Pull request by mseaton**
…tend

## Issue I worked on
see https://openmrs.atlassian.net/browse/SDK-371

### CODE REVIEW

**Review by ibacher — APPROVED** [review]
Thanks @mseaton! Looks great! I agree this is a much simpler

**Comment by ibacher** [comment]
Build failure is my fault...

File: sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/ContentHelper.java [hunk 1]
```diff
@@ -201,28 +174,44 @@ List<File> installFrontendConfig(ContentPackage contentPackage, Map<String, Stri
...
     }
 
+    /**
+     * Copies the contents of the sourceDir into the targetDir.
+     * If namespace is passed in, files are added to a namespaced subdirectory within each copied directory
+     */
+    void copyDirectory(File sourceDir, File targetDir, String namespace) throws IOException {
+        boolean emptyNamespace = StringUtils.isBlank(namespace) || namespace.equals(".") || namespace.equals("/") || namespace.equals("false");
+        if (emptyNamespace) {
+            log.debug("Copying " + sourceDir + " to " + targetDir);
```
[the remark this answers is not in the document]
Comment (mseaton) [reply sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/ContentHelper.java]: Done.  I'm generally bad about this.

**Comment by mseaton** [comment]
> Build failure is my fault...

No worries @ibacher .  I recently added some code to be less dependent on the hiv content package for other testing.  I'll remove the dependency on the hiv content package.

**Pull request merged**